### DISCUSSION TITLE
Support component alias for storybook search

### DISCUSSION THREAD

**Discussion by BoppLi** [SchwarzIT/onyx#1292] [Ideas]
We might look if we can support alternative names of components for the storybook search, e.g. searching for "dropdown" could result in the "Select"
![image](https://github.com/SchwarzIT/onyx/assets/151019977/bfc452ce-3716-494b-8a5b-777eaf187e12)
![image](https://github.com/SchwarzIT/onyx/assets/151019977/cb499836-c77e-44f5-b1d2-ab441cf1917d)

This might simplify finding components, especially when migrating from other component libs (like CoreUI)

**Comment by mj-hof** [comment]
Storybook doesn't offer any feature to cover this. But definetly a cool idea that we will maybe try to achieve with our own implementation
